products/unity.md

```diff
@@ -17,8 +17,8 @@ releases:
 -   releaseCycle: "6"
     releaseDate: 2024-04-29
     eol: false
-    latest: "6000.0.47f1"
-    latestReleaseDate: 2025-04-16
+    latest: "6000.1.0f1"
```

Comment:
According to https://unity.com/blog/unity-6-1-is-now-available this is unity 6.1, which is a separate release than 6.0. See also https://unity.com/releases/unity-6/support.

Note that the description may need to be updated too.

Reply:
But their download page put it on 6.x release part. So if i understand it correctly it eoled 6000.0  branch
You can observe that from : https://unity.com/releases/editor/archive

Reply:
I doubt that, 6.0 is an LTS ans still listed on https://unity.com/releases/editor/archive under the LTS tab.

Reply:
nope 6 series is LTS not 6.0

Reply:
maybe i'm wrong maybe i misjudged it

Reply:
so do you suggest to open a new cycle for non lts 6.x parts ?

Reply:
let me do that give me 2 min

Reply:
now it looks much better ( I have seperated 6 series branches for LTS versioned and nonLTS ones